cfme/services/catalogs/orchestration_template.py

```diff
@@ -207,7 +187,38 @@ def exists(self):
             return False
 
 
-@navigator.register(OrchestrationTemplate, 'All')
+@attr.s
+class OrchestrationTemplateCollection(BaseCollection):
+    """A collection for the :py:class:`cfme.services.catalogs.orchestration_template`"""
+    ENTITY = OrchestrationTemplate
+
+    def instantiate(self, template_type, template_name, description, draft, content):
+        return self.ENTITY.from_collection(self, template_type, template_name,
+                                           description, draft, content)
+
+    def create(self, template_name, description, template_type, draft=None, content=None):
+        self.template_type = template_type
```

Comment:
**QUESTION**
What's the purpose of this?

Reply:
template type object is required here
https://github.com/ManageIQ/integration_tests/pull/6857/files#diff-b320dec2a69c9f5ff2599b8ef491dde5R248.